docker-compose.dev.yml

```diff
@@ -2,3 +2,14 @@ version: "3.7"
 services:
   app:
     build: .
+  seed_marquez_db:
+    build: ./web/docker/db
+    environment:
+      - MARQUEZ_HOST=app
+      - MARQUEZ_PORT=5000
+      - MARQUEZ_ADMIN_PORT=5001
+    volumes:
+      - ./docker/wait-for-it.sh:/usr/local/bin/wait-for-it.sh
+    depends_on:
+      - app
+    entrypoint: [ "./wait-for-it.sh", "app:5000", "--", "./entrypoint.sh" ]
```

Comment:
I decided to put the seed step here for now which will add the seed data when we use the `docker/up.sh -- build` command to start the project. We could drive this process with a different approach as well.